### DISCUSSION TITLE
Add customisable website dark theme...

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2343] [Ideas] [answered]
...Why not?
And with that, could you please move the colour customisation settings from 2.0-3.0 to this?
Oh, and can you please just call them "Website dark theme" and "Editor dark theme" and not include the "with customisable colours" part?

**Comment by cobaltt7** [comment, marked as answer]
Duplicate of #645

**Comment by mxmou** [comment]
> Oh, and can you please just call them "Website dark theme" and "Editor dark theme" and not include the "with customisable colours" part?

Depending on how you configure it, it isn't necessarily a dark theme.
